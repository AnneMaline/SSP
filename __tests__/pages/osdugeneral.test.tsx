import { render, screen } from "@testing-library/react";
import OSDUGeneralPage from "../../src/app/OSDU-general/page";

describe("OSDUGeneral Page", () => {
  it("renders correctly", () => {
    const { container } = render(<OSDUGeneralPage />);
    expect(container).toMatchSnapshot();
  });
});
