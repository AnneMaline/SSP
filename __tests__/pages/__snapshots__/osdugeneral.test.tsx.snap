// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OSDUGeneral Page renders correctly 1`] = `
<div>
  <div>
    <h1>
      OSDU general
    </h1>
  </div>
</div>
`;
